pkg/webui/console/views/device-general-settings/network-server-form/validation-schema.js

```diff
@@ -255,6 +255,13 @@ const validationSchema = Yup.object()
               return Yup.number().strip()
             }
 
+            if (mode === ACTIVATION_MODES.ABP) {
+              return Yup.number()
+                .min(0, Yup.passValues(sharedMessages.validateNumberGte))
+                .max(15, Yup.passValues(sharedMessages.validateNumberLte))
+                .required(sharedMessages.validateRequired)
```

Comment:
You should additionally also conditionally set the `required` prop on the field component itself.